docs/UserGuideCS2101.md

```diff
@@ -0,0 +1,424 @@
+* The index refers to the index number shown in the displayed student list.
+* The index must be a positive integer 1, 2. 3, ….
+
+Example:
+
+* `helpstu 2`
+
+### Deleting a student: `deletestu`
+
+Removes a specific student.
+If a student has left the module or swapped tutorial group, you can delete him from the list. This feature is hardly 
+used.
+
+Format: `deletestu INDEX`
+
+* Deletes the student at the specified INDEX.
+* The index refers to the index number shown in the displayed student list.
+* The index must be a positive integer 1, 2. 3, ….
+
+Example:
+
+* `deletestu 2`
+
+### Finding a student: `findstu`
+
+Finds one or more specific students.
+As the student list can potentially contain more than 40 students, this feature helps you to specifically find the 
+student based on the student's name.
+
+Format: `findstu KEYWORD [MORE_KEYWORDS]`
+
+* Finds student(s) with specified KEYWORD [MORE_KEYWORDS] in their name(s).
+* The keyword is case-insensitive.
+
+Examples:
+
+* `findstu bob`
+* `findstu john mary`
+
+### Listing all students: `liststu`
+
+Lists all students in the student list.
+After the above feature(`findstu`) is used, you may want to view all the students that you have again. This feature 
+helps you to do so.
+
+Format: `liststu`
+
+Example:
+
+* `liststu`
+
+### Questions
+Sometimes, you may receive questions that you may not have the answer to or you may not have enough time to answer 
+all of them during the tutorial. SETA allows you to track these questions. Furthermore, if you find a question that 
+requires urgent clarification or critical thinking, you can indicate on the user interface that these questions are 
+important. For questions requiring critical thinking, it would be also good to share them with other TAs so that the 
+rest of the module's students can benefit from knowing this question.
+
+### Adding a question : `addq`
+
+Adds a question to the question list.
+This feature allows you to keep track of the students' questions.
+
+Format: `addq QUESTION_DESCRIPTION`
+
+Example:
+
+* `addq what is the difference between self-invocation and call-backs for sequence diagrams?`
+
+### Marking a question : `markq`
+
+Marks a question as important.
+This feature enables you to indicate on the user interface that this question is important and requires more 
+attention compared to the other questions.
+
+Format: `markq INDEX`
+
+* Marks the question at the specified INDEX as important.
+* The index refers to the index number shown in the displayed question list.
+* The index must be a positive integer 1, 2, 3, ...
+* The index must be within the number of questions in the question list. E.g. There are 4 questions. The possible
+  indexes are 1, 2, 3 and 4.
+
+Example:
+
+* `markq 1` marks the first question in the question list as important
+
+### Unmarking a question : `unmarkq`
+
+Marks a question as unimportant. (If the question was previously or mistakenly marked as important)
```

Comment:
Allows you to mark a question as unimportant if the question was previously or mistakenly marked as important

Reply:
done